src/llvm.rs

```diff
@@ -128,12 +128,13 @@ fn handle_module(
                 "faerie define",
             );
             // TODO: reloc should derive from Copy
-            for &(ref reloc, ref external_name, offset) in &compilation.relocs.relocs {
+            for &(ref reloc, ref external_name, offset, _addend) in &compilation.relocs.relocs {
                 // FIXME: What about other types of relocs?
                 // TODO: Faerie API: Seems like it might be inefficient to
                 // identify the caller by name each time.
                 // TODO: Faerie API: It's inconvenient to keep track of which
                 // symbols are imports and which aren't.
+                // TODO: How should addend be handled? Should it be added to offset here?
```

Comment:
You can pass the added to faerie below, replacing the `0` in the `addend: 0` below, which should do everything needed.